source/client/factories_impl.cc

```diff
@@ -168,18 +170,43 @@ RequestSourceFactoryImpl::create(const Envoy::Upstream::ClusterManagerPtr& clust
   for (const auto& option_header : request_options.request_headers()) {
     setRequestHeader(*header, option_header.header().key(), option_header.header().value());
   }
-
   if (!options_.requestSource().empty()) {
     RELEASE_ASSERT(!service_cluster_name.empty(), "expected cluster name to be set");
     // We pass in options_.requestsPerSecond() as the header buffer length so the grpc client
     // will shoot for maintaining an amount of headers of at least one second.
     return std::make_unique<RemoteRequestSourceImpl>(cluster_manager, dispatcher, scope,
                                                      service_cluster_name, std::move(header),
                                                      options_.requestsPerSecond());
+  } else if (options_.requestSourcePluginConfig().has_value()) {
+    absl::StatusOr<RequestSourcePtr> plugin_or = LoadRequestSourcePlugin(
+        options_.requestSourcePluginConfig().value(), api_, std::move(header));
+    // GCOVR_EXCL_START
```

Comment:
I didn't expect the GCOVR exclusion here, but rather at/around the RELEASE_ASSERT line above -- so, sorry could you remove it here? Looking at the [coverage report](https://22476-180498819-gh.circle-artifacts.com/0/coverage/source/client/factories_impl.cc.gcov.html) everything is fine now, including the RELEASE_ASSERT line, so it seems we don't need it there either now (probably because there's no conditional argument being passed).

Reply:
Sounds good.